Makefile

```diff
@@ -82,9 +82,13 @@ install: | $(ACTIVATE_SCRIPT)
 
 .PHONY: install
 
-test-like-ci: | $(ACTIVATE_SCRIPT)
-	${ACTIVATE} .github/actions/download-and-run-tests/install-test-requirements.py
-	${ACTIVATE} .github/actions/download-and-run-tests/run-tests.py
+TEST_LIKE_CI_RUN_SH := venv/actions/includes/actions/python/run-installed-tests/run.sh
+$(TEST_LIKE_CI_RUN_SH):
+	if [ ! -d venv/actions ]; then git clone https://github.com/SymbiFlow/actions venv/actions; fi
+	cd venv/actions && git pull
```

Comment:
Should there be a make target to update `.github/workflows/install-and-test.yml` ?

Reply:
Already there - see https://github.com/SymbiFlow/symbiflow-rr-graph/pull/16/files/ebd02e5dc8b92f03abe9ea812fed2b626e27599b#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R105-R114

Reply:
Should probably have a `.PHONY` target, rather than requiring users to type full path.  Also README or README.developers doesn't mention invoking the target?

Reply:
Moved to https://github.com/SymbiFlow/symbiflow-rr-graph/pull/17